code/src/index.css

```diff
@@ -1,13 +1,245 @@
 body {
   margin: 0;
+  margin-bottom: 150px;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color:#000;
+  text-decoration: none;
+}
+
+html {
+  position: relative;
+  min-height: 100%;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* Movie cards styling*/
+
+.movie-cards-container { 
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  flex-flow: wrap;
+}
+
+.movie-card {
+  flex-basis: 25%;
+}
+
+ .movie-card img {
+   width: 100%;
+   margin: 0;
+}
+.overlay-container {
+  position: relative;
+}
+
+.text-overlay { 
+  position: absolute;
+  bottom: 0;
+  color: #fff;
+  background: rgba(0, 0, 0, 0.742);
+  width: 100%;
+  height: 100%;
+  transition: 0.3s ease;
+  opacity: 0;
+}
+
+.text-overlay h1 { 
+  position: absolute;
+  bottom: 50px;
+  margin-left: 20px;
+  font-size: 30px;
+}
+
+.text-overlay p {
+  position: absolute;
+  bottom: 0;
+  padding: 20px;
+  font-size: 17px;
+}
+
+.text-overlay:hover {
+  opacity: 1;
+}
+
+/* Movie pages Styling */
+.movie-page { 
```

Comment:
for "movie card styling" you have both a space above and below the note. pick one and stick with it all the way through your document.